action/test_action.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Exit immediately if a command returns a non-zero status.
+set -e
+
+# Assert we are running in docker
+if [ ! -f /.dockerenv ]; then
+    exit 1
+fi
+
+# Check basic tools
+java -version
+rbenv -v
+adb version
```

Comment:
aren't we using gcloud instead of adb to run large tests ? 🤔

Reply:
@AchrafAmil We only testing here the Docker Container Action, which use the Docker Android image. So here we are just testing the wrapping is ok, not the image itself.